### PR TITLE
trunk-5134

### PR DESCRIPTION
**Pull request by Parthis1833**
## Description of what I change

i have changed path using File.sperator to java's paths.

## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-5134

## Checklist: I completed these to help reviewers :)

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [x] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [x] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: api/src/main/java/org/openmrs/util/UpgradeUtil.java [hunk 1]
```diff
@@ -41,9 +42,12 @@ public static Integer getConceptIdForUnits(String units) {
 		String appDataDir = OpenmrsUtil.getApplicationDataDirectory();
 		Properties props = new Properties();
 		String conceptId = null;
-		String filePath = appDataDir +
-				System.getProperty("file.separator") +
-				DatabaseUtil.ORDER_ENTRY_UPGRADE_SETTINGS_FILENAME;
+//		String filePath = appDataDir +
```
Comment (tendomart) [inline api/src/main/java/org/openmrs/util/UpgradeUtil.java:45]: @Parthis1833 please remove commented out code

Comment (Parthis1833) [reply api/src/main/java/org/openmrs/util/UpgradeUtil.java:45]: Ok

**Comment by tendomart** [comment]
Give the Ticket a good title, something like, 

**TRUNK-5134:Replace path constructions using separator char**

**Comment by tendomart** [comment]
Also everytime you work on an issue, hit the claim issue button so other developers may not work on it as you working on it same time

**Comment by Parthis1833** [comment]
Ok sir

**Comment by tendomart** [comment]
When done with changes , go back to the Ticket and hit the `Request Code Review` button

**Comment by Parthis1833** [comment]
Ok sir

**Comment by dkayiwa** [comment]
@Parthis1833 did you see the maven build failures?

**Comment by Parthis1833** [comment]
yes sir i  see that

**Comment by dkayiwa** [comment]
Are you fixing them?

**Comment by Parthis1833** [comment]
sir I need help in my system  project properly running and build but why here  it's showing an error

**Comment by dkayiwa** [comment]
You can ask for help on Talk.

**Comment by tendomart** [comment]
@Parthis1833, @dkayiwa   he means you post your problem at [talk.openmrs.org](https://talk.openmrs.org/)

**Comment by dkayiwa** [comment]
@Parthis1833 did you take a look at the merge conflicts?

**Comment by Parthis1833** [comment]
Sorry sir I try too slove that  error but it can be slove  it I try what I can  .

**Pull request closed without merging**